great-idea-website/index.html

```diff
@@ -16,30 +16,38 @@
 
   <body>
 
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
-
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+ <nav> 
+   <a href="#">Services</a>
+   <a href="#"> Product</a>
+   <a href="#">Vision</a>
+   <a href="#">Features</a>
+   <a href="#"> About</a>
+   <a href="#"> Contact</a>
+
+   <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+ </nav> 
+<header>
+  <h1>
   Innovation
   On
   Demand
+  </h1>
+ 
 
-  Get Started
+  <button<>Get Started</button>
```

Comment:
Careful with typos! Javascript is a really lax language so it allows this through, but be careful.